src/mon/OSDMonitor.cc

```diff
@@ -10956,7 +10957,19 @@ bool OSDMonitor::prepare_pool_op_create(MonOpRequestRef op)
 int OSDMonitor::_check_remove_pool(int64_t pool_id, const pg_pool_t& pool,
 				   ostream *ss)
 {
-  const string& poolstr = osdmap.get_pool_name(pool_id);
+  const std::string& poolstr = osdmap.get_pool_name(pool_id);
+
+  // Check permission
+  if (!g_conf->mon_allow_pool_delete) {
+    *ss << "pool deletion is disabled; you must first set the "
+        << "mon_allow_pool_delete config option to true before you can "
+        << "destroyi a pool";
```

Comment:
destroyi?

Reply:
What @amitkumar50 said. Also, please have `mon_allow_pool_delete` between apostrophes, so it is clearer what the config option name is, as well as `true`, as the value to be set to. Bonus points if you end the statement with a period. :)